src/ComposableController.ts

```diff
@@ -36,45 +22,22 @@ export class ComposableController extends BaseController<any, any> {
    * @param controllers - Map of names to controller instances
    * @param initialState - Initial state keyed by child controller name
    */
-  constructor(controllers: ControllerList = [], initialState?: any) {
-    super();
+  constructor(controllers: ControllerList) {
+    super(
+      undefined,
+      controllers.reduce((state, controller) => {
+        state[controller.name] = controller.state;
+        return state;
+      }, {} as any),
+    );
     this.initialize();
-    this.cachedState = initialState;
     this.controllers = controllers;
-    this.cachedState = undefined;
-  }
-
-  /**
-   * Get current list of child composed store instances
-   *
-   * @returns - List of names to controller instances
-   */
-  get controllers() {
```

Comment:
Note that the getter and setter for the `controllers` property has been removed. It appears to have been completely unused. I searched and found no references to it in this repo or in mobile.